src/foam/core/Constant.js

```diff
@@ -38,7 +38,7 @@ foam.CLASS({
     {
       name: 'value',
       expression: function(factory) {
-        return factory();
+        return factory ? factory() : null;
```

Comment:
I'd be curious to know when this happens, are people setting factory to null?

Reply:
I triggered it when I had a constant that only had a swiftFactory and no js values.